LSP.sublime-settings

```diff
@@ -314,6 +314,13 @@
         "Packages/Elixir/Syntaxes/Elixir.tmLanguage",
       ]
     },
+    "erlang-ls": {
+      "command"   : [ "/path/to/my/erlang_ls", "--transport", "stdio" ],
+      "enabled"   : true,
```

Comment:
Please remove `"enabled": true` here.

Other than that, looks mergeable to me!